package.json

```diff
@@ -50,7 +51,7 @@
     "transform": {
       ".(ts|tsx)": "<rootDir>/node_modules/ts-jest/preprocessor.js"
     },
-    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$",
+    "testRegex": "(/__tests__/.*|/*|\\.(test|spec))\\.(ts|tsx|js)$",
```

Comment:
@Benglin This is to include nested folders in "__tests__" folder while running tests.  I've fixed a bug in this line in last commit in this PR.